src/pykoala/corrections/flux_calibration.py

```diff
@@ -26,21 +26,47 @@
 
 class FluxCalibration(CorrectionBase):
     """
-    This module contains the methods for extracting and applying an absolute flux calibration.
-
-    Description
-    -----------
-
+    A class to handle the extraction and application of absolute flux calibration.
 
     Attributes
     ----------
+    name : str
+        The name of the Correction.
+    verbose : bool
+        If True, prints additional information during execution.
+    calib_spectra : None or array-like
+        The calibration spectra data.
+    calib_wave : None or array-like
+        The calibration wavelength data.
+    response : None or array-like
+        The response data.
+    resp_wave : None or array-like
+        The response wavelength data.
+    response_units : float
+        Units of the response function, default is 1e16 (erg/s/cm^2/AA).
```

Comment:
I have the feeling that the only attribute that should be a class attribute is `name`. All the others should be instance attributes, initialised in the constructor:
<https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables>

Reply:
Right. I made the changes.